tests/ModelTest.php

```diff
@@ -488,9 +492,56 @@ public function testUnset(): void
         $this->assertTrue(isset($user2->note2));
 
         $user2->unset(['note1', 'note2']);
+        $user2->save();
```

Comment:
I believe there was a separate issue where the model was not marked as dirty - maybe assert on `isDirty` as well?

Reply:
It was not dirty since the change was done directly in the database.
I added the assertions on `isDirty` for exhautivity.